apps/SageAccountWeb/src/components/RegisterAccount1.tsx

```diff
@@ -33,7 +33,7 @@ import {
   StyledInnerContainer,
   StyledOuterContainer,
 } from './StyledComponents'
-import { DEFAULT_SOURCE_APP_ID, useSourceApp } from './useSourceApp'
+import { SYNAPSE_SOURCE_APP_ID, useSourceApp } from './useSourceApp'
```

Comment:
Deleted DEFAULT_SOURCE_APP_ID, and now using SYNAPSE_SOURCE_APP_ID everywhere for clarity (they were both constants set to 'synapse.org')